docs/guides/security-policy-for-authors.md

```diff
@@ -431,7 +421,7 @@ Please see the software documentation for further information.
 
 ## License and use of this document
 
-* Version: 0.2.3
+* Version: 0.3.0
```

Comment:
Let‘s be a good example and bump the version to 1.0.0 as it‘s not a draft any more!